Compilation error in Vulkano on target `aarch64-linux-android`

# Setup

*   Version of vulkano: `0.25.0`
*   OS: `Windows 10 build 19043.1165`
*   Target: `aarch64-linux-android`

# Issue

The `vulkano` crate of version `0.25.0` cannot be built on target `aarch64-linux-android` due to compilation errors.

### Command
```
cargo build --target aarch64-linux-android
```

### Command line output

```
   Compiling vulkano v0.25.0
error[E0053]: method `from_vulkan` has an incompatible type for trait
   --> C:\Users\timur\.cargo\registry\src\github.com-1ecc6299db9ec823\vulkano-0.25.0\src\device\properties.rs:215:25
    |
140 |     fn from_vulkan(val: F) -> Option<Self>;
    |                         - type in trait
...
215 |     fn from_vulkan(val: [i8; N]) -> Option<Self> {
    |                         ^^^^^^^
    |                         |
    |                         expected `u8`, found `i8`
    |                         help: change the parameter type to match the trait: `[u8; N]`
    |
    = note: expected fn pointer `fn([u8; N]) -> Option<_>`
               found fn pointer `fn([i8; N]) -> Option<_>`

error[E0308]: mismatched types
   --> C:\Users\timur\.cargo\registry\src\github.com-1ecc6299db9ec823\vulkano-0.25.0\src\device\properties.rs:216:38
    |
216 |         Some(unsafe { CStr::from_ptr(val.as_ptr()).to_string_lossy().into_owned() })
    |                                      ^^^^^^^^^^^^ expected `u8`, found `i8`
    |
    = note: expected raw pointer `*const u8`
               found raw pointer `*const i8`

error: aborting due to 2 previous errors

Some errors have detailed explanations: E0053, E0308.
For more information about an error, try `rustc --explain E0053`.
error: could not compile `vulkano`

To learn more, run the command again with --verbose.
```

--- a/vulkano/src/device/properties.rs
+++ b/vulkano/src/device/properties.rs
@@ -93,7 +93,7 @@ impl<const N: usize> FromVulkan<[f32; N]> for [f32; N] {
 
 impl<const N: usize> FromVulkan<[std::os::raw::c_char; N]> for String {
     #[inline]
-    fn from_vulkan(val: [i8; N]) -> Option<Self> {
+    fn from_vulkan(val: [std::os::raw::c_char; N]) -> Option<Self> {
         Some(unsafe { CStr::from_ptr(val.as_ptr()).to_string_lossy().into_owned() })
     }
 }

